plugins/woocommerce-admin/client/marketing/overview-multichannel/types.tsx

```diff
@@ -0,0 +1,11 @@
+export type Plugin = {
+	slug: string;
+	icon: string;
+	name: string;
+	description: string;
+	status: string;
```

Comment:
❓  Maybe plugin.status could be an enum? That also could help in `getButton()` in order to use `plugin.status.CONFIGURED` instead of using the 'configured' string ?

https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Thanks. Instead of enums, I think a better and simpler approach would be [string literal unions](https://www.typescriptlang.org/docs/handbook/literal-types.html#string-literal-types). More info: p7H4VZ-3sK-p2

I'll look into this.

Reply:
Done in commit [`acae609` (#34080)](https://github.com/woocommerce/woocommerce/pull/34080/commits/acae609a7da4c5d59d21ed51b9dbf1150c4542f0).

Reply:
Interesting! Thanks :til: